docs/geedocs/5.2.3/answer/7160003.html

```diff
@@ -39,7 +39,7 @@ <h2>Release notes: Open GEE 5.2.3</h2>
 <p>Open GEE 5.2.3 is currently in development</p>
 
   <h5>
-    <p id="overview_5">New Features</p>
+    <p id="overview_5.2.3">New Features</p>
```

Comment:
There are a number of new features and a bug that have been implemented for 5.2.3 that I don't see represented in the release notes: https://github.com/google/earthenterprise/milestone/4?closed=1

Please add these